GeeksCoreLibrary/Components/Configurator/Services/ConfiguratorsService.cs

```diff
@@ -845,6 +1247,21 @@ public async Task<string> ReplaceConfiguratorItemsAsync(string template, VueConf
             {
                 return template;
             }
+            
+            // Replace external configurator ID if one is provided.
+            if (configuration.ExternalConfiguration != null && !String.IsNullOrWhiteSpace(configuration.ExternalConfiguration.Id))
+            {
+                if (!isDataQuery)
+                {
+                    template = template.Replace($"{{externalConfiguration.id}}", configuration.ExternalConfiguration.Id);
```

Comment:
Why do you use string interpolation here, without actually interpolating anything? Now you have to escape the brackets for no reason. Please change this line to this:
```
template = template.Replace("{externalConfiguration.id}", configuration.ExternalConfiguration.Id);
```